### ISSUE TITLE
此项目bert encoding是不是耗时比较长？

### ISSUE THREAD

**Issue by Lotus-Blue** [RUCAIBox/RecBole-PJF#14]
![image](https://github.com/RUCAIBox/RecBole-PJF/assets/45590660/ea897ea1-8891-46bc-9493-efb1873c967d)
老师您好，我用的colab在线跑的，T4，每次进行bert encoding都好慢，2m文件差不多得编码1个小时，是我哪里出错了吗？还是说硬件不行？期待老师回复，非常感谢~

**Comment by Lotus-Blue** [comment]
是因为这是在CPU上运行的原因吗

**Comment by Lotus-Blue** [comment]
打扰了，确实是用的cpu，我改成了用gpu，快了很多
![image](https://github.com/RUCAIBox/RecBole-PJF/assets/45590660/795bd55f-a2a5-451f-ae89-c7e43420071e)

1、导入CUDA相关包:
可以在文件开头导入:
import torch.cuda

2、设置device:
可以在__init__()方法里面添加:
def __init__(self, config):
  ...

  self.device = torch.device("cuda")

3、初始化BERT模型时指定device:
self.model = AutoModel.from_pretrained(MODEL_PATH).to(self.device)

4、输入数据到GPU:
在bert_encoding()方法中,在模型输入前:

input_ids = input['input_ids'].to(self.device)
attention_mask = input['attention_mask'].to(self.device)

outputs = self.model(input_ids, attention_mask=attention_mask)

5、 输出结果回CPU:
在bert_encoding()方法的最后:

output = outputs[0][:, 0].cpu().detach()
return output

**Comment by Lotus-Blue** [comment]
thanks
